src/main/java/com/technocredits/orangeHRM/pages/LoginPage.java

```diff
@@ -1,21 +1,33 @@
 package com.technocredits.orangeHRM.pages;
 
-public class LoginPage {
-
-	private static LoginPage loginPage;
-
-	private LoginPage() {
-
+import java.io.IOException;
+import java.util.Properties;
+
+import com.technocredits.orangeHRM.base.PredefinedMethods;
+import com.technocredits.orangeHRM.util.PropertyFileReader;
+
+public class LoginPage extends PredefinedMethods{
+	private static LoginPage loginPage = null;
+	Properties loginPageProperties;
+	
+	private LoginPage() throws IOException{
```

Comment:
Add comment like what this method is doing.

Reply:
Making constructor as private for LoginPage, so no one would be able to create object for LoginPage directly. We want only 1 single instance to be created. Creating object from getInstance() method.